src/modules/location/sagas.ts

```diff
@@ -37,8 +37,22 @@ function* handleSetENSContentSuccess(action: SetENSContentSuccessAction) {
   }
 }
 
-function* goToActivity() {
-  yield put(replace(locations.activity()))
+function* goToActivity(action: ClaimNameSuccessAction | null = null) {
```

Comment:
maybe rename this? so its not called goToActivity?

Reply:
makes sense, I'll rename it